src/components/link-preview/link-preview.pw.tsx

```diff
@@ -0,0 +1,266 @@
+/* eslint-disable jest/valid-expect, no-unused-expressions */
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import LinkPreviewComponentTest from "./components.test-pw";
+import { LinkPreviewProps } from ".";
+import {
+  linkPreview,
+  linkPreviewCloseIcon,
+  linkPreviewAs,
+} from "../../../playwright/components/link-preview";
+import {
+  checkGoldenOutline,
+  checkAccessibility,
+} from "../../../playwright/support/helper";
+import { CHARACTERS } from "../../../playwright/support/constants";
+import { HooksConfig } from "../../../playwright";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const testCypress = "test-cypress";
+const urlProp = "./carbon-by-sage-logo.png";
+const keysToTrigger = ["Space", "Enter"];
+
+test.describe("when focused", () => {
```

Comment:
All the test should not come under this describe block so could you please make it separate describe block